modPrecompiled/test/org/aion/precompiled/KeccakHashTest.java

```diff
@@ -0,0 +1,52 @@
+package org.aion.precompiled;
```

Comment:
should be moved to `package org.aion.precompiled.contracts`; also missing license header